sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -116,6 +116,23 @@ class DataSourceV2SQLSuite
       Row("data", "string", "")))
   }
 
+  test("DescribeNamespace using v2 catalog") {
+    spark.sql("CREATE NAMESPACE IF NOT EXISTS testcat.ns1.ns2 COMMENT " +
+      "'test namespace' LOCATION '/tmp/ns_test'")
+    val descriptionDf = spark.sql("DESCRIBE NAMESPACE testcat.ns1.ns2")
+    assert(descriptionDf.schema.map(field => (field.name, field.dataType)) ===
+      Seq(
+        ("name", StringType),
+        ("value", StringType)
+      ))
+    val description = descriptionDf.collect()
+    assert(description === Seq(
+      Row("Namespace Name", "ns2"),
+      Row("Description", "test namespace"),
+      Row("Location", "/tmp/ns_test")
+    ))
+  }
+
```

Comment:
Can you also remove the following TODO: (https://github.com/apache/spark/blob/ab981f10a61ac15be14334b015fe2206e088314f/sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala#L793)